dev/CommandBarFlyout/InteractionTests/TextCommandBarFlyoutTests.cs

```diff
@@ -668,6 +668,54 @@ public void VerifyTextCommandBarRemainsOpenWithItems()
             }
         }
 
+        [TestMethod]
+        public void ValidateSelectionFlyoutDoesNotTakeFocus()
+        {
+            using (var setup = new TextCommandBarFlyoutTestSetupHelper())
+            {
+                // RichEditBox only implements the Text pattern, which the "TextBlock" MITA type exposes.
+                var richEditBox = new TextBlock(FindElement.ById("RichEditBox"));
+                var fillWithTextButton = new Button(FindElement.ById("RichEditBoxFillWithTextButton"));
+
+                Log.Comment("Give focus to the RichEditBox.");
+                FocusHelper.SetFocus(richEditBox);
+
+                Log.Comment("Enter enough text to guarantee that double-tapping the RichEditBox will select text.");
+                fillWithTextButton.InvokeAndWait();
+
+                Log.Comment("Double-tap to select the text and bring up the selection menu. The CommandBarFlyout should appear, but should not take focus.");
+                InputHelper.LeftDoubleClick(richEditBox);
+                Wait.ForIdle();
+
+                var boldButton = FindElement.ByName("Bold");
+                Verify.IsNotNull(boldButton);
+                Verify.IsFalse(boldButton.HasKeyboardFocus);
+
+                Log.Comment("Press backspace to delete the selected text. This should work because the RichEditBox should still has focus.");
```

Comment:
has --> have

Reply:
D'oh, that's what happens when I reword things after the fact. :)